extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/VertxOAuth2AuthenticationMechanism.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.oidc;
+package io.quarkus.oidc.runtime;
```

Comment:
Is this class supposed to be exposed to the users? It looks like an internal one but I would like to check.